pkgs/tools/security/gnupg/23.nix

```diff
@@ -34,6 +34,8 @@ stdenv.mkDerivation rec {
     ./tests-add-test-cases-for-import-without-uid.patch
     ./allow-import-of-previously-known-keys-even-without-UI.patch
     ./accept-subkeys-with-a-good-revocation-but-no-self-sig.patch
+    ./0001-agent-Fix-signing-messages-over-475-bytes.patch
```

Comment:
Have these been sent upstream? If so, please link to the submission.

Reply:
I believe @puckipedia has attempted to send these upstream, but their mailing list never picked up the messages. I can't upstream any of these myself, since I expect they'll hassle me about my name.

Reply:
> > Have these been sent upstream? If so, please link to the submission.
> I believe @puckipedia has attempted to send these upstream, but their mailing list never picked up the messages. I can't upstream any of these myself, since I expect they'll hassle me about my name.

The other patches don't have a link to submission, and one of them doesn't apply on 2.4.0. See update to 2.4.0 at https://github.com/NixOS/nixpkgs/pull/207071 .